dusty/systems/docker/__init__.py

```diff
@@ -65,10 +65,8 @@ def _get_dusty_containers(client, services, include_exited=False):
     of services. If no services are provided, attempts to
     return all containers associated with Dusty."""
     if services:
-        return [container
-                for container in client.containers(all=include_exited)
-                if any('/{}'.format(get_dusty_container_name(service)) in container.get('Names', [])
-                       for service in services)]
+        containers = [_get_container_for_app_or_service(client, service, include_exited=include_exited) for service in services]
+        return [container for container in containers if container]
```

Comment:
The difference is that there's better handling in the case where there's no such container.  So it will print nice stuff to the user if they typo instead of silently just skipping the container that's not found